cpp/log/cluster_state_controller-inl.h

```diff
@@ -348,8 +348,9 @@ void ClusterStateController<Logged>::OnServingSthUpdated(
     }
 
     actual_serving_sth_.reset(new ct::SignedTreeHead(update.handle_.Entry()));
-    LOG(INFO) << "Received new Serving STH:\n"
-              << actual_serving_sth_->DebugString();
+    LOG(INFO) << "Received new Serving STH: timestamp: "
+              << actual_serving_sth_->timestamp()
+              << " tree_size: " << actual_serving_sth_->tree_size();
```

Comment:
It's gonna be quite a long line, but yeah, you just know that we'll not have the bit we need!
Done.